src/components/Admin/AddPopUpClinic/AdminPopUpClinic.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from "react";
+import { Card, Form, FormLayout, TextField } from "@shopify/polaris";
+import { useTranslation } from "react-i18next";
+import { postalCodeIsValid } from "../../../utils/postalCode";
+import { phoneNumberIsValid } from "../../../utils/phoneNumber";
+
+export interface IPopUpClinic {
+  name: string;
+  date: string;
+  street: string;
+  city: string;
+  province: string;
+  postalCode: string;
+  phone: string;
+  websiteUrl: string;
+  [key: string]: string;
+}
+
+const handleSubmit = () => {
+  return true;
+};
+
+export function AdminPopUpClinic() {
+  const { t } = useTranslation();
+
+  const [popUpClinic, setPopUpClinc] = useState<IPopUpClinic>({
+    name: "",
+    date: new Date().toDateString(),
+    street: "",
+    city: "",
+    province: "",
+    postalCode: "",
+    phone: "",
+    websiteUrl: "",
+  });
+
+  const [popUpClinicErrors, setPopUpClincErrors] = useState<IPopUpClinic>({
```

Comment:
All this state is derived from the state in `poUpClinic` - it doesn't need to be stored, I think it just needs to be derived when needed.